From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jason Perlow <jperlow@gmail.com>
Date: Mon, 15 Jun 2026 15:10:00 -0400
Subject: [PATCH] drm/cix: update drm_color_format in dp_component for v7.1

Linux v7.1 changed DRM_COLOR_FORMAT_* to DRM_OUTPUT_COLOR_FORMAT_*.
Update the CIX hw/dp_component.c references accordingly.
---
diff --git a/drivers/gpu/drm/cix/linlon-dp/hw/dp_component.c b/drivers/gpu/drm/cix/linlon-dp/hw/dp_component.c
index a12c45449950..3ad436b7fc95 100644
--- a/drivers/gpu/drm/cix/linlon-dp/hw/dp_component.c
+++ b/drivers/gpu/drm/cix/linlon-dp/hw/dp_component.c
@@ -1179,11 +1179,11 @@ static void dp_improc_update(struct linlondp_component *c,
 		IPS_CTRL_CHD420 | IPS_CTRL_DITH;
 
 	/* config color format */
-	if (st->color_format == DRM_COLOR_FORMAT_YCBCR420)
+	if (st->color_format == DRM_OUTPUT_COLOR_FORMAT_YCBCR420)
 		ctrl |= IPS_CTRL_YUV | IPS_CTRL_CHD422 | IPS_CTRL_CHD420;
-	else if (st->color_format == DRM_COLOR_FORMAT_YCBCR422)
+	else if (st->color_format == DRM_OUTPUT_COLOR_FORMAT_YCBCR422)
 		ctrl |= IPS_CTRL_YUV | IPS_CTRL_CHD422;
-	else if (st->color_format == DRM_COLOR_FORMAT_YCBCR444)
+	else if (st->color_format == DRM_OUTPUT_COLOR_FORMAT_YCBCR444)
 		ctrl |= IPS_CTRL_YUV;
 
 	/* config dither */
@@ -1265,12 +1265,12 @@ static int dp_improc_init(struct dp_dev *dp, struct block_header *blk,
 
 	improc = to_improc(c);
 	improc->supported_color_depths = BIT(8) | BIT(10);
-	improc->supported_color_formats = DRM_COLOR_FORMAT_RGB444 |
-					  DRM_COLOR_FORMAT_YCBCR444 |
-					  DRM_COLOR_FORMAT_YCBCR422;
+	improc->supported_color_formats = DRM_OUTPUT_COLOR_FORMAT_RGB444 |
+					  DRM_OUTPUT_COLOR_FORMAT_YCBCR444 |
+					  DRM_OUTPUT_COLOR_FORMAT_YCBCR422;
 	value = linlondp_read32(reg, BLK_INFO);
 	if (value & IPS_INFO_CHD420)
-		improc->supported_color_formats |= DRM_COLOR_FORMAT_YCBCR420;
+		improc->supported_color_formats |= DRM_OUTPUT_COLOR_FORMAT_YCBCR420;
 
 	improc->supports_csc = true;
 	improc->supports_gamma = true;
